projects/js-packages/publicize-components/src/hooks/use-dismiss-notice/index.js

```diff
@@ -2,6 +2,13 @@ import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
 import apiFetch from '@wordpress/api-fetch';
 import { useCallback, useMemo, useState } from 'react';
 
+const calculateReappearanceTime = minutes => {
```

Comment:
Like we chatted earlier, let's change this to seconds and the param could passed could be `secondsToSnooze`